cluster-autoscaler/cloudprovider/huaweicloud/huaweicloud_service_manager.go

```diff
@@ -251,7 +251,7 @@ func (csm *cloudServiceManager) ListScalingInstances(groupID string) ([]huaweicl
 			return nil, fmt.Errorf("unexpected get nil scaling group instances")
 		}
 
-		klog.Info("Got %d instances from scaling group, total instances: %d", len(*response.ScalingGroupInstances), *response.TotalNumber)
```

Comment:
What's the log look like? I mean the previous one.

Reply:
like:
```go
Got %d instances from scaling group, total instances: %d10 50
```